phpBB/phpbb/console/command/update/check.php

```diff
@@ -0,0 +1,311 @@
+	* @param OutputInterface 		$output		Output stream, used to print messages.
+	* @param OutputInterface		$stability	Force a given stability
+	* @param bool					$recheck	Disallow the use of the cache
+	* @param string					$ext_name	The extension name
+	* @return int
+	*/
+	protected function check_ext(InputInterface $input, OutputInterface $output, $stability, $recheck, $ext_name)
+	{
+		$ext_manager = $this->phpbb_container->get('ext.manager');
+		$md_manager = $ext_manager->create_extension_metadata_manager($ext_name, null);
+		$updates_available = $ext_manager->version_check($md_manager, $recheck, false, $stability);
+
+		$metadata = $md_manager->get_metadata('all');
+		if ($input->getOption('verbose'))
+		{
+			$output->writeln('<info>' . $md_manager->get_metadata('display-name') . '</info>');
+			$output->writeln('');
+
+			$output->writeln('<comment>' . $this->user->lang('CURRENT_VERSION') . $this->user->lang('COLON') . '</comment> ' . $metadata['version']);
+		}
+
+		if (!empty($updates_available))
+		{
+			$output->writeln('');
+			$output->writeln('<question>' . $this->user->lang('NOT_UP_TO_DATE', $metadata['name']) . '</question>');
+
+			if ($input->getOption('verbose'))
+			{
+				$this->display_versions($output, $updates_available);
+			}
+
+			return 1;
+		}
+		else
+		{
+			$output->writeln('');
+			$output->writeln('<question>' . $this->user->lang('NOT_UP_TO_DATE', $metadata['name']) . '</question>');
+
+			if ($input->getOption('verbose'))
+			{
+				$output->writeln('<info>' . $this->user->lang('UPDATE_NOT_NEEDED') . '</info>');
+			}
+
+			return 0;
+		}
+	}
+
+	/**
+	* Check if the core is up to date
+	*
+	* @param InputInterface 		$input 		Input stream, used to get the options.
+	* @param OutputInterface 		$output		Output stream, used to print messages.
+	* @param OutputInterface		$stability	Force a given stability
+	* @param bool					$recheck	Disallow the use of the cache
+	* @return int
+	*/
+	protected function check_core(InputInterface $input, OutputInterface $output, $stability, $recheck)
+	{
+		$version_helper = $this->phpbb_container->get('version_helper');
+		$version_helper->force_stability($stability);
+
+		$updates_available = $version_helper->get_suggested_updates($recheck);
+
+		if ($input->getOption('verbose'))
+		{
+			$output->writeln('<info>phpBB core</info>');
+			$output->writeln('');
+
+			$output->writeln('<comment>' . $this->user->lang('CURRENT_VERSION') . $this->user->lang('COLON') . '</comment> ' . $this->config['version']);
+		}
+
+		if (!empty($updates_available))
+		{
+			$output->writeln('');
+			$output->writeln('<question>' . $this->user->lang('UPDATE_NEEDED') . '</question>');
+
+			if ($input->getOption('verbose'))
+			{
+				$this->display_versions($output, $updates_available);
+			}
+
+			return 1;
+		}
+		else
+		{
+			if ($input->getOption('verbose'))
+			{
+				$output->writeln('');
+				$output->writeln('<question>' . $this->user->lang('UPDATE_NOT_NEEDED') . '</question>');
```

Comment:
I think this should always be displayed, even if verbose is disabled. Right now this just returns empty.